tck/graphs/named-graphs.adoc

```diff
@@ -67,3 +67,8 @@ For label/sublabel combinations, the supported advice is:
 
 * `implies`
 ** This indicates that the existence of the label on a node implies the existence of the sublabel. In other words, if a node has the label, it will always have the sublabel.
+
+=== Scripts
+
+The metadata file will specify one or more script files, which are files that contain Cypher statements which are to be used to create the named graph.
```

Comment:
I'll fix the two grammar things. On advice, I'm inclined to perhaps leave it out altogether, if not now perhaps later, so I don't want to spend too much thinking on it.